packages/webpack/src/presets/style.ts

```diff
@@ -97,6 +97,10 @@ function createCssLoadersRule (ctx: WebpackConfigContext, cssLoaderOptions) {
 
   if (options.webpack.extractCSS) {
     if (ctx.isServer) {
+      // https://webpack.js.org/loaders/css-loader/#exportonlylocals
+      if (cssLoader.options.modules) {
+        cssLoader.options.modules.exportOnlyLocals = true
+      }
```

Comment:
maybe also check that the user hasn't overridden this option?
```suggestion
      if (cssLoader.options.modules) {
        cssLoader.options.modules.exportOnlyLocals = cssLoader.options.modules.exportOnlyLocals ?? true
      }
```

Reply:
I think it makes no sense, because in this case it should be true. Otherwise the class names will not be applied in SSR.